remix-analyzer/src/solidity-analyzer/index.ts

```diff
@@ -1,25 +1,31 @@
 'use strict'
 import { AstWalker } from 'remix-astwalker'
 import list from './modules/list'
+import { CompilationResult, AnalyzerModule, AnalysisReportObj, AnalysisReport } from 'types'
+
+type ModuleObj = {
+  name: string
+  mod: AnalyzerModule
+}
 
 export default class staticAnalysisRunner {
 
-  run (compilationResult, toRun, callback) {
-    const modules = toRun.map((i) => {
-      const m = this.modules()[i]
+  run (compilationResult: CompilationResult, toRun: any[], callback: ((reports: AnalysisReport[]) => void)): void {
+    const modules: ModuleObj[] = toRun.map((i) => {
```

Comment:
when I look at the code, it seems that `toRun` is a number[]. Could you type it & comment it ? It's difficult to understand what it is exactly

Reply:
updated